android/build.gradle

```diff
@@ -148,9 +163,9 @@ dependencies {
 
 if (isNewArchitectureEnabled()) {
     react {
-        reactRoot = rootProject.file("../node_modules/react-native/")
+        reactNativeDir = rootProject.file(findNodeModulePath(rootProject.rootDir, "react-native") ?: "../node_modules/react-native/")
```

Comment:
What's the reason for changing `reactRoot` to `reactNativeDir`?

Reply:
reactRoot is deprecated, see https://github.com/facebook/react-native/pull/33142